{
  try {
    when(operationResult.canContinue()).thenThrow(new RuntimeException("force a failure"));
    buildCauseProducerService.manualSchedulePipeline(Username.ANONYMOUS,pipelineConfig.name(),new ScheduleOptions(),operationResult);
    fail("expected exception, got none");
  }
 catch (  Exception e) {
    assertThat(triggerMonitor.isAlreadyTriggered(CaseInsensitiveString.str(pipelineConfig.name())),is(false));
  }
}
